hnn_core/parallel_backends.py

```diff
@@ -518,6 +518,10 @@ def simulate(self, net, n_trials, postproc=False):
             connected.
         n_trials : int
             Number of trials to simulate.
+        tstop : float
+            The simulation stop time (ms).
+        dt : float
+            The integration time step of h.CVode (s)
```

Comment:
This is not correct: the integration time step is given in _milliseconds_, just like the rest of our times. This needs to be correct in three places (two here and one in `dipole.py`.

Reply:
good catch! I don't know why in my head I was equating this with the interval at which the print outputs were happening :)